node/core/dispute-coordinator/src/real/mod.rs

```diff
@@ -633,10 +633,14 @@ async fn handle_import_statements(
 	now: Timestamp,
 	pending_confirmation: oneshot::Sender<ImportStatementsResult>,
 	metrics: &Metrics,
-) -> Result<ImportStatementsResult, Error> {
+) -> Result<(), Error> {
 	if state.highest_session.map_or(true, |h| session + DISPUTE_WINDOW < h) {
 		// It is not valid to participate in an ancient dispute (spam?).
-		return Ok(ImportStatementsResult::InvalidImport)
+		pending_confirmation
+			.send(ImportStatementsResult::InvalidImport)
+			.map_err(|_| Error::OneshotSend)?;
+
+		return Ok(())
```

Comment:
I think that's reverting @drahnr changes. He changed it so, that `hande_import_statements` would just return the result and the caller can take care of answering the oneshot.

Reply:
Uhm - yeah the commit even says "revert" :1st_place_medal:  ... why not just fix it?

Reply:
cherry picked #3963

Reply:
I think those changes actually caused it to fail, since I forgot that the default feature set does not include `"disputes"` which rendered the CI kind of useless on that PR #3944